benchmarks/smallmatvec/manual_versions/libxsmm/results/results_h32_vertical.plot

```diff
@@ -0,0 +1,168 @@
+set key maxcolumns 0 maxrows 0
+set key noopaque
+unset label
+unset arrow
+set style increment default
+unset style line
+unset style arrow
+set style histogram clustered gap 2 title textcolor lt -1
+unset object
+set style textbox transparent margins  1.0,  1.0 border  lt -1 linewidth  1.0
+set offsets 0, 0, 0, 0
+set pointsize 1
+set pointintervalbox 1
+set encoding default
+unset polar
+unset parametric
+unset decimalsign
+unset micro
+unset minussign
+set view 60, 30, 1, 1
+set view azimuth 0
+set rgbmax 255
+set samples 100, 100
+set isosamples 10, 10
+set surface 
+unset contour
+set cntrlabel  format '%8.3g' font '' start 5 interval 20
+set mapping cartesian
+set datafile separator whitespace
+unset hidden3d
+set cntrparam order 4
+set cntrparam linear
+set cntrparam levels auto 5
+set cntrparam points 5
+set size ratio 0 1,1
+set origin 0,0
+set style data points
+set style function lines
+unset xzeroaxis
+unset yzeroaxis
+unset zzeroaxis
+unset x2zeroaxis
+unset y2zeroaxis
+set xyplane relative 0.5
+set tics scale  1, 0.5, 1, 1, 1
+set mxtics default
+set mytics default
+set mztics default
+set mx2tics default
+set my2tics default
+set mcbtics default
+set mrtics default
+set nomttics
+set xtics border in scale 1,0.5 mirror norotate  autojustify
+set xtics  norangelimit autofreq 
+set ytics border in scale 1,0.5 mirror norotate  autojustify
+set ytics  norangelimit autofreq 
+set ztics border in scale 1,0.5 nomirror norotate  autojustify
+set ztics  norangelimit autofreq 
+unset x2tics
+unset y2tics
+set cbtics border in scale 1,0.5 mirror norotate  autojustify
+set cbtics  norangelimit autofreq 
+set rtics axis in scale 1,0.5 nomirror norotate  autojustify
+set rtics  norangelimit autofreq 
+unset ttics
+set title "Matvec benchmark, increasing levels, 32 threads, 2*16-core skylake, 32*32 horizontal"
+set title  font "" norotate
+set timestamp bottom 
+set timestamp "" 
+set timestamp  font "" norotate
+set trange [ * : * ] noreverse nowriteback
+set urange [ * : * ] noreverse nowriteback
+set vrange [ * : * ] noreverse nowriteback
+set xlabel "vertical levels" 
+set xlabel  font "" textcolor lt -1 norotate
+set x2label "" 
+set x2label  font "" textcolor lt -1 norotate
+set xrange [ * : * ] noreverse nowriteback
+set x2range [ * : * ] noreverse nowriteback
+set ylabel "time (s)" 
+set ylabel  font "" textcolor lt -1 rotate
+set y2label "" 
+set y2label  font "" textcolor lt -1 rotate
+set yrange [ * : * ] noreverse nowriteback
+set y2range [ * : * ] noreverse nowriteback
+set zlabel "" 
+set zlabel  font "" textcolor lt -1 norotate
+set zrange [ * : * ] noreverse nowriteback
+set cblabel "" 
+set cblabel  font "" textcolor lt -1 rotate
+set cbrange [ * : * ] noreverse nowriteback
+set rlabel "" 
+set rlabel  font "" textcolor lt -1 norotate
+set rrange [ * : * ] noreverse nowriteback
+unset logscale
+unset jitter
+set zero 1e-08
+set lmargin  -1
+set bmargin  -1
+set rmargin  -1
+set tmargin  -1
+set locale "en_GB.UTF-8"
```

Comment:
$ gnuplot results_h32_vertical.plot 

This line caused this script to fail for me:

    set locale "en_GB.UTF-8"
                            ^
    "results_h32_vertical.plot", line 151: Locale not available

Reply:
OK, I've removed it from all of the files. It should just pick up the default one instead.